tox.ini

```diff
@@ -45,13 +45,55 @@ deps =
     twcurrent: Twisted
     twtrunk: https://github.com/twisted/twisted/archive/trunk.zip
 
+    {trial,coverage}: mock
+
+    coverage: coverage
+
+
+whitelist_externals =
+    coverage: cp
+    coverage: mkdir
+
+
+passenv =
+    PATH
+    LANG LC_ALL
+    CI CONTINUOUS_INTEGRATION
+    TRAVIS TRAVIS_* _system_*
+    CODECOV_OPTIONS
+
+
+setenv =
+    PIP_DISABLE_PIP_VERSION_CHECK=1
+
+    coverage: COVERAGE_FILE={toxworkdir}/coverage/coverage.{envname}
+    {coverage_combine,codecov}: COVERAGE_FILE={toxworkdir}/coverage/coverage
+
+    {coverage,coverage_combine}: COVERAGE_HTML={envlogdir}/coverage_report_html
+    {coverage,coverage_combine,codecov}: COVERAGE_XML={envlogdir}/coverage_report.xml
+
+
 commands =
     "{toxinidir}/.travis/environment"
-    coverage run -p "{envdir}/bin/trial" {posargs:klein}
+
+    # Run trial without coverage
+    trial: trial --random=0 --logfile="{envlogdir}/trial.log" --temp-directory="{envlogdir}/trial.d" {posargs:klein}
+
+    coverage: mkdir -p "{toxworkdir}/coverage"
+    coverage: coverage run --rcfile="{toxinidir}/.coveragerc" "{envdir}/bin/trial" --random=0 --logfile="{envlogdir}/trial.log" --temp-directory="{envlogdir}/trial.d" {posargs:klein}
```

Comment:
That's just what the file has always been called.  I don't have any trouble finding it, but I'm happy to rename it.